pkg/operator/obj/webhooktriggerdeps.go

```diff
@@ -241,15 +242,26 @@ func (wtd *WebhookTriggerDeps) AnnotateTriggerToken(ctx context.Context, target
 	}
 
 	annotations := wtd.WebhookTrigger.Object.GetAnnotations()
+	labels := wtd.WebhookTrigger.Object.GetLabels()
 
 	claims.KubernetesNamespaceName = wtd.TenantDeps.Namespace.Name
 	claims.KubernetesNamespaceUID = string(wtd.TenantDeps.Namespace.Object.GetUID())
 	claims.KubernetesServiceAccountToken = sat
 
+	// FIXME This annotation is not being set right now ...
 	claims.RelayDomainID = annotations[model.RelayDomainIDAnnotation]
-	claims.RelayTenantID = annotations[model.RelayTenantIDAnnotation]
+
+	// FIXME This annotation is not being set right now ...
+	if tenant, ok := annotations[model.RelayTenantIDAnnotation]; ok {
+		claims.RelayTenantID = tenant
+	} else {
+		claims.RelayTenantID = strings.TrimPrefix(labels[model.RelayControllerTenantNameLabel], "workflow-")
+	}
+
 	claims.RelayName = mt.Name
-	idh.Set("parents", claims.RelayDomainID, claims.RelayTenantID)
+	claims.RelayTriggerID = labels[model.RelayWorkflowTriggerIDLabel]
+
+	idh.Set("parents", claims.RelayDomainID, claims.RelayTenantID, claims.RelayTriggerID)
```

Comment:
Do we need to add this? I don't think the trigger sync ID can change without the entire object being replaced, can it?

Reply:
It was accomplishing nothing before, as this data wasn't being set.
I'm not sure any of these can justifiable change without it referencing a different object.

Reply:
The argument for including the trigger sync ID is the same as these other two items. They shouldn't change, but if they do, we should probably update the token. Although I'd probably be fine with removing this altogether as well.